composer.json

```
@@ -37,9 +37,9 @@
         "symfony/http-kernel": "^5.4 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "doctrine/coding-standard": "^12",
-        "phpstan/phpstan": "^2",
-        "phpunit/phpunit": "^10.5.38 || ^11"
+        "doctrine/coding-standard": "12.0.0",
+        "phpstan/phpstan": "2.1.8",
+        "phpunit/phpunit": "^10.5.38 || 11.4.14"
```

```suggestion
        "phpunit/phpunit": "^10.5.38 || ^11.4.14"
```

Was it necessary to not use a version range?

---

Yes, see af9f8c7